$OpenBSD: patch-configure,v 1.1.1.1 2008/12/11 23:03:11 jasper Exp $

Prevent errors like this during configure:
GtkWarning: could not open display

--- configure.orig	Wed Dec  3 00:18:42 2008
+++ configure	Wed Dec  3 00:19:26 2008
@@ -6679,23 +6679,7 @@ fi
 v=`echo 0.9.0 | sed -e "s/\./,/g"`
 { echo "$as_me:$LINENO: checking for 0.9.0 of goocanvas" >&5
 echo $ECHO_N "checking for 0.9.0 of goocanvas... $ECHO_C" >&6; }
-prog="
-import goocanvas
-ver = goocanvas.pygoocanvas_version
-
-if type(ver) == str:
-	ver = ver.split('.')
-	ver = map(int,ver)
-	ver = tuple(ver)
-elif type(ver) == tuple:
-	pass
-else:
-	raise Exception
-
-if ver >= ($v):
-	print 'yes'
-"
-out=`$PYTHON -c "$prog"`
+out="yes"
 if test "x$out" = xyes; then
   { echo "$as_me:$LINENO: result: yes" >&5
 echo "${ECHO_T}yes" >&6; }
